industrial_extrinsic_cal/include/industrial_extrinsic_cal/camera_yaml_parser.h

```diff
@@ -0,0 +1,51 @@
+#ifndef CAMERA_YAML_PARSER_H
+#define CAMERA_YAML_PARSER_H
+
+#include <boost/shared_ptr.hpp>
+#include <yaml-cpp/yaml.h>
+#include <iostream>
+#include <industrial_extrinsic_cal/basic_types.h>
+#include <industrial_extrinsic_cal/camera_definition.h>
+#include <industrial_extrinsic_cal/transform_interface.hpp>
+#include <industrial_extrinsic_cal/trigger.h>
+
+
+namespace industrial_extrinsic_cal {
+
+  using std::string;
```

Comment:
Typically we don't use `using` in a header.  This pollutes any file that might import the header.  These are "scoped" so maybe it's not an issue.  Thoughts?